### DISCUSSION TITLE
About the optimization of openraft

### DISCUSSION THREAD

**Discussion by caicancai** [databendlabs/openraft#954] [Q&A] [answered]
Hello, openraft seems to be a very interesting project. I would like to ask if there are any papers or documents about openraft's optimization of raft.

**Comment by caicancai** [comment]
@drmingdrmer hello, excuse me. I know a few people who have this problem, and maybe we can explain it in this disscussion, although I'm ready to look at the code to explore it🤔

**Reply by drmingdrmer** [reply]
What questions do they have? 
We can just have a discussion here or in the discord channel:
https://discord.gg/ZKw3WG7FQ9

**Reply by caicancai** [reply]
thank you

**Comment by drmingdrmer** [comment, marked as answer]
No paper yet. However, you can find some of my informal pieces on my blog:
https://blog.openacid.com/
https://github.com/drmingdrmer/consensus-essence
